api/v1alpha1/database_types.go

```diff
@@ -27,7 +27,8 @@ import (
 
 // DatabaseSpec defines the desired state of Database
 type DatabaseSpec struct {
-	NDB      NDB      `json:"ndb"`
+	// +kubebuilder:validation:Required
```

Comment:
Is it possible to move this to webhooks as well?

Reply:
No, since this is a required field irrespective of other values, annotation based validations are sufficient for this.